worldengine/simulations/irrigation.py

```diff
@@ -11,22 +10,42 @@ def execute(self, world, seed):
 
     @staticmethod
     def _calculate(world):
+        #Notes on performance:
+        #  -method is run once per generation
+        #  -iterations        : width * height
+        #  -memory consumption: width * height * 8 Byte (numpy 1.9.2)
+
```

Comment:
This is nice, I love documentation and I love precise memory allocation prediction and measurements. :)